Dockerfile

```diff
@@ -49,29 +50,36 @@ ENV PATH /usr/local/bin:$PATH
 RUN cd /usr/local/bin && \
   ln -s $(which python3) python
 
-# install python dependencies
-RUN pip install -r requirements.txt && \
-  useradd -lm kivy
-
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     locale-gen
 ENV LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
 
+# Install Android SDK
+ENV ANDROID_HOME=/opt/android
+ENV ANDROIDSDK=$ANDROID_HOME/sdk
+ENV ANDROIDNDK=$ANDROIDSDK/ndk-bundle
```

Comment:
Again, acknowledging GH commit messages:
https://github.com/learningequality/kolibri-installer-android/pull/107/commits/a74373b6cd811f58519762afcac5fb5e143755f4
https://github.com/learningequality/kolibri-installer-android/pull/107/commits/edcd28b85034730d1041ead505ec9654147fa5d6

I don't see an issue with the `ndk-bundle` softlink, but I'll defer to @rtibbles

I do want to point out a downside of including these definitions here: multiple sources of truth. If there need to be any changes to the envars there are now 2 places to update, which defeats the spirit of going through Makefile (not saying that we're flawless in this regard now though)

Reply:
I kind of see it the other way. When you run the Makefile normally, it requires that you have `ANDROID_HOME`, `ANDROIDSDK` and `ANDROIDNDK` setup in the environment already. Having those environment variables embedded in the image means that's already taken care of in the container. It makes the image very analogous to the [Ubuntu Github runners](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#environment-variables-3).

Reply:
The only reason this stuff isn't _only_ in the Dockerfile, but also referenced in the Makefile is to allow local devs to leverage the same tooling outside of a container, I think a little bit of repetition here is fine.

Reply:
If @rtibbles is cool with it, I have no objections!